infrastructure/worker.tf

```diff
@@ -64,6 +64,8 @@ resource "aws_iam_role_policy" "worker_task_execution_role_policy" {
           "${aws_ssm_parameter.crossfeed_send_db_name.arn}",
           "${data.aws_ssm_parameter.db_username.arn}",
           "${data.aws_ssm_parameter.db_password.arn}",
+          "${data.aws_ssm_parameter.pe_db_username.arn}",
+          "${data.aws_ssm_parameter.pe_db_password.arn}",
```

Comment:
Add pe variables to this section: https://github.com/cisagov/crossfeed/blob/df31606935754fc6e63db58a800833b6747b6e6e/infrastructure/worker.tf#L199-L206